public/js/mathjax-config-extra.js

```diff
@@ -4,5 +4,8 @@ window.MathJax = {
   tex2jax: {
     inlineMath: [['$', '$'], ['\\(', '\\)']],
     processEscapes: true
+  },
+  asciimath2jax: {
+    delimiters: [['$', '$'], ['\\(', '\\)']]
```

Comment:
I'm not sure they can be the same as tex2jax :/

I just checked the official default config and they are different there.
https://github.com/mathjax/MathJax/blob/master/config/default.js#L392-L398

By the way, my test is this:

```
sum_(i=1)^n i^3=((n(n+1))/2)^2
```

From http://asciimath.org/ which also recommends mathjax